File: apps/institution-api/Models/MigrationLog.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class MigrationLog
    {
        public Guid MigrationId { get; set; }
        public string ScriptChecksum { get; set; }
        public string ScriptFilename { get; set; }
        public DateTime CompleteDt { get; set; }
        public string AppliedBy { get; set; }
        public byte Deployed { get; set; }
        public string Version { get; set; }
        public string PackageVersion { get; set; }
        public string ReleaseVersion { get; set; }
        public int SequenceNo { get; set; }
    }
}

File: libs/common/models/migration-log.ts



export class MigrationLog {
    public migrationId: string = '';
    public scriptChecksum: string = '';
    public scriptFilename: string = '';
    public completeDt: string = '';
    public appliedBy: string = '';
    public deployed: number = 0;
    public version: string = '';
    public packageVersion: string = '';
    public releaseVersion: string = '';
    public sequenceNo: number = 0;

    constructor(init? : Partial<MigrationLog>) {
        
        Object.assign(this, init);
    }
}

